### ISSUE TITLE
Minimal working example

### ISSUE THREAD

**Issue by maflcko** [krobelus/clang-diff-playground#1]
Hey, it would be helpful to mention a minimal working example for `clang-diff` in the readme or docs. For example, a diff between two hello_world.cpp files.

**Comment by krobelus** [comment]
yeah, it would enable people to see some examples without having to compile and play around with it. I mainly used this repo for the report though. The code itself is in the clang repo.

anyway, here's an example

a.cpp
```cpp
#include <stdio.h>

void a () {
  if (0) {
    printf("hello, world\n");
  }
}
```
b.cpp
```cpp
#include <stdio.h>

void b () {
  printf("hello, world\n");
}
```

```bash
$ clang-diff /tmp/{a,b}.cpp --
Update TypeLoc(2)
Move CompoundStmt(4) into FunctionDecl(1) at 1
Delete CompoundStmt(4)
Delete IfStmt(5)
Delete IntegerLiteral(6)
```
html diff:
`clang-diff -html -s=200 -dump-matches test/Tooling/Inputs/clang-diff-basic-src.cpp test/Tooling/clang-diff-basic.cpp -- -std=c++11`
